Lesson23/Mesut/map.js

```diff
@@ -215,6 +215,11 @@ console.log('Exercise 5', studentsExercise5Result2);
 // Given an array of numbers, convert each number to a string representation using .map().
 const numbersExercise6 = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
 
+const numbersExercise6Result =numbersExercise6.map(string=>string.toString())
```

Comment:
Since there's an array of number, giving a number (element of array) name "string" is misleading

```suggestion
const numbersExercise6Result =numbersExercise6.map(number => number.toString())
```